src/chart/line/LineView.js

```diff
@@ -354,7 +354,7 @@ export default ChartView.extend({
         // FIXME step not support polar
         var step = !isCoordSysPolar && seriesModel.get('step');
         var clipShapeForSymbol;
-        if (coordSys && coordSys.getArea) {
+        if (coordSys && coordSys.getArea && seriesModel.get('clip')) {
```

Comment:
It's more robust to use `seriesModel.get('clip', true)`.

In this way it won't get the `clip` property in the root `option` if it is not exists in `series`.